model/flow/node_version_info.go

```diff
@@ -0,0 +1,20 @@
+package flow
```

Comment:
@peterargue ,yes, there was a circular dependency when trying to keep the `NodeVersionInfo` and `CompatibleRange` structs in the access module while separating the conversion methods into the `engine/common/rpc/convert` module.

I guess we could consider one of the following approaches:
1) Keep both the structs and conversion methods together in the `access` module (similar to how it's done with `TransactionResult`).
https://github.com/onflow/flow-go/blob/6701e9e74d3d4518ce383ba1bc55b8bd756fe3d8/access/api.go#L213-L235
2) Move the structs to the `flow` module and keep the conversion methods in the `engine/common/rpc/convert` module (as it is in the current implementation).

Reply:
let's do 1 for now, and we can circle back to refactor these properly. I don't want to add scope creep to this PR, but also think it's best to fix everything at once since it will require updates to other repos

Reply:
[50dcd73](https://github.com/onflow/flow-go/pull/6294/commits/50dcd73fdd3f6530d354f84bebe71d2348ed0717)